### PR TITLE
Add packages needed to authenticate to gmail

### PR DESCRIPTION
**Pull request by Pelado-Mat**
Needed to correctly authenticate using a gmail relay

### CODE REVIEW

**Comment by bokysan** [comment]
What kind of issues are you running into without these packages, for future reference?

File: Dockerfile [hunk 1]
```diff
@@ -33,6 +33,7 @@ RUN        true && \
            apk add --no-cache opendkim && \
            apk add --no-cache ca-certificates tzdata supervisor rsyslog && \
            apk add --no-cache --upgrade musl musl-utils && \
+           apk add --no-cache --upgrade  cyrus-sasl cyrus-sasl-plain && \
```
Comment (bokysan) [inline Dockerfile]: We should probably squish lines 34-36 to make the building process faster?

**Comment by Pelado-Mat** [comment]
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA512

Hi, 
    I got the same errors referred in this post:

https://stackoverflow.com/questions/20238922/postfix-sasl-authentication-failure-no-worthy-mechs-found


Boky <notifications@github.com> wrote:
> What kind of issues are you running into without these
> packages, for future reference?
>

- -- 
Sent using Mailpile, Free Software from www.mailpile.is

-----BEGIN PGP SIGNATURE-----

iQGcBAEBCgAGBQJcb3WDAAoJEAypDzjVQLnsehoL/1HY7pLE76VjY+8ytBDoNqRj
QPFwAVhwQu/OyEHZaLPtpaKSRhtyE/U6h0brXPUnji7wZMWXsYPcWdnj4pYy5WSX
jRawNtUUa4beOReI5nu5RvWWnazBvbSgqrZrUHfI9Zige5b4ELM7FyqxoQrXTWNl
Zbgql5V+a30nIoHVKSZ6ceiS4cn2EXoUc/5BybYuoJeuyZe056Zd0PUJLO3QTbc2
tJlWBKzhM/dehzbh9b+C6xhxd8l2Qf0A9ObHSfgICQbg2N/9VSGqzREmXoUIbsVY
FpsJwWW+vVu6DNWZd8uk3b4V+d+r4D6Am/x3FuMrYjVp01+Y8Mobe4iqvaXkJRYS
mk4d/dtLM3eHy4jSjx9X7U548jnERzRZcm0PWBj8m+A3mz5zzWnMjjtDDKLxabKZ
kTj8kJL09YOxkrH5FqSSABTwWjXMpbiSzFtp72l2gaNeB0bfYexvWjLaYQYEO7+9
4gTrhEbTDCroQgrR26Qd5KrX5bDaeW2t7WGiVOZwfQ==
=Mn2N
-----END PGP SIGNATURE-----

**Comment by bokysan** [comment]
Thank you for your commit!

**Pull request merged**